app/parse.py

```diff
@@ -15,8 +34,103 @@ class Product:
     num_of_reviews: int
 
 
+class ChromeDriver:
+    def __init__(self) -> None:
+        self._driver = webdriver.Chrome()
```

Comment:
Consider adding exception handling when initializing the Chrome WebDriver. This can help manage scenarios where the WebDriver might not be available or fails to start.